itemManagement/models.py

```diff
@@ -40,6 +39,12 @@ def totalQuantity(self):
         # return sum(location.count for location in self.locations.all())
         return self.locations.aggregate(Sum('itemstorage__quantity'))['itemstorage__quantity__sum'] or 0
 
+    @property
+    def needToNotifyAdmin(self):
```

Comment:
This can be return self.alertThreshold is not none and self.totalQuantity <= alertThreshold